web/src/components/common/GroupBuyMerchantHeader.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+
+import GPayLogo from 'assets/gpay-logo.svg';
```

Comment:
I think we should consider importing SVGs as React Components so we can use them as-is instead of rendering them as images. You can do so by:
```
import { ReactComponent as GPayLogo } from 'assets/gpay-logo.svg';
```
And you could use it directly like `<GPayLogo />` without the image tag

Reply:
done!